src/Expansion.php

```diff
@@ -362,8 +404,33 @@ private function expanderExpansion() : Parser {
             (
                 ns()->as('expander')
                 ,
-                either(parentheses(), braces())->as('args')
+                either(
+                    chain(token('('), $this->expanderAstExpansion(), token(')')) // recursion !!!
+                    ,
+                    chain(token('('), layer()->as('layer_arg'), token(')'))
+                    ,
+                    chain(token('{'), braces()->as('layer_arg'), token('}'))
```

Comment:
Whoops, this should have been `chain(token('{'), layer()->as('layer_arg'), token('}'))`